intera_examples/scripts/gripper_keyboard.py

```diff
@@ -84,6 +95,7 @@ def offset_holding(offset):
                                        key=lambda x: x[1][2]):
                     print("  %s: %s" % (key, val[2]))
     # force shutdown call if caught by key handler
+    gripper.set_dead_zone(original_deadzone)
```

Comment:
This should probably happen in clean_shutdown too. If someone Ctrl-C's to quit, as they do in every other example, these lines will not get hit.